src/views/Project/index.jsx

```diff
@@ -366,33 +356,16 @@ export default class Project extends Component {
       [...issues, ...goodFirstBugs, ...mentoredBugs],
       'summary'
     );
-    const { introductionOpen } = this.state;
 
     return (
       <Dashboard title={project.name}>
         {project.introduction && (
-          <Card>
-            <CardContent>
-              <Typography>
-                <Markdown
-                  source={
-                    introductionOpen
-                      ? project.introduction
-                      : project.introduction.split(/(?=##\s)/)[0]
-                  }
-                  renderers={{ link: this.linkRenderer }}
-                />
-              </Typography>
-            </CardContent>
-            <CardActions>
-              <Button
-                size="small"
-                onClick={this.handleButtonClick}
-                className={classes.seeMoreButton}>
-                {introductionOpen ? 'See Less' : 'See More'}
-              </Button>
-            </CardActions>
-          </Card>
+          <ExpansionCard>
```

Comment:
1. Rename `ExpansionCard` to `ProjectIntroductionCard`.
2. Add a prop `introduction` to `ProjectIntroductionCard` so that we can render the markdown directly from `ProjectIntroductionCard`.